stingray/pulse/pulsar.py

```diff
@@ -411,20 +411,28 @@ def z2_n_detection_level(n=2, epsilon=0.01, ntrial=1):
     ----------------
     ntrial : int
         The number of trials executed to find this profile
+    n_summed_spectra : int
+        Number of Z_2^n periodograms that are being averaged
 
     Returns
     -------
     detlev : float
         The epoch folding statistics corresponding to a probability
         epsilon * 100 % that the signal has been produced by noise
     """
-    if ntrial > 1:
-        simon("Z2_n: The treatment of ntrial is very rough. Use with caution")
+    try:
+        from scipy import stats
+    except:  # pragma: no cover
+        raise Exception('You need Scipy to use this function')
```

Comment:
pretty sure scipy is a required dependency for stingray, so this statement would be inconsisent with our requirements?

Reply:
Yes, I'll take care of it